generate

```diff
@@ -92,3 +92,7 @@ rm ./protocol-resilience-test/llcupdate1/src/main/java/module-info.java
 
 # customization
 autorest --version=3.6.6 --use:. customization-tests/swagger
+
+# partial update tests
+autorest --version=3.6.6 --use:. partial-update-tests/existing/swagger/README.md --handle-partial-update
```

Comment:
suggestion, move `--handle-partial-update` to your swagger/readme?

Reply:
updated